azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/ConfigMojo.java

```diff
@@ -543,7 +586,10 @@ private static WebAppConfiguration getConfigurationFromExisting(WebApp webapp, A
             builder.os(OperatingSystemEnum.Docker);
         } else if (webapp.operatingSystem() == OperatingSystem.LINUX) {
             builder.os(OperatingSystemEnum.Linux);
-            builder.runtimeStack(AppServiceUtils.parseRuntimeStack(webapp.linuxFxVersion()));
+            final RuntimeStack runtimeStack = AppServiceUtils.parseRuntimeStack(webapp.linuxFxVersion());
+            builder.runtimeStack(runtimeStack);
```

Comment:
`RuntimeStack` should contains the java info, do we need to set `javaVersion` in builder?